packages/eslint-config-widen/src/typescript.ts

```diff
@@ -19,7 +19,8 @@ export = {
         '@typescript-eslint/no-unused-vars': [
           'error',
           {
-            argsIgnorePattern: '^_',
+            ignoreRestSiblings: true,
+            varsIgnorePattern: '^_',
```

Comment:
I ended up having to add argsIngorePattern back into our project to ignore underscores, so this might not be working?

Reply:
Relevant? Not sure what you were seeing that triggered the change. Could also be our setup in upload... we're updating dependencies right now. https://stackoverflow.com/questions/64052318/how-to-disable-warn-about-some-unused-params-but-keep-typescript-eslint-no-un